test/form/module-name-scoped-package/_expected/umd.js

```diff
@@ -0,0 +1,13 @@
+(function (global, factory) {
+	typeof exports === 'object' && typeof module !== 'undefined' ? factory(exports) :
+	typeof define === 'function' && define.amd ? define(['exports'], factory) :
+	(factory((global['@scoped/npm-package'] = global['@scoped/npm-package'] || {})));
```

Comment:
allow arbitrary strings, including npm scoped packages.  Not sure this is/should be a requirement, but I added a test anyway because of #582